cmd/trayscale/app.go

```diff
@@ -221,7 +221,7 @@ func (a *App) update(s tsutil.Status) {
 	if a.online != online {
 		a.online = online
 		a.notify(online) // TODO: Notify on startup if not connected?
-		systray.SetIcon(statusIcon(online))
+		a.tray.SetIcon(online)
```

Comment:
`a.tray` might be `nil` here.

Reply:
Anticipating this, I have deliberately decided to run `if t == nil` inside `SetIcon`. Clients don't have to care, the implementation handles the rest.